packages/eslint-plugin/lib/config/workleap.ts

```diff
@@ -0,0 +1,19 @@
+import type { Linter } from "eslint";
+import StrictCSSModuleNames from "../rules/strict-css-modules-names.ts";
+
+const config: Linter.FlatConfig =
```

Comment:
What are the drawbacks of importing the rule directly in the index file like it was done previously?

Reply:
I believe I was avoiding a circular dependency by moving it to its own file